connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -977,4 +977,10 @@ public interface ConnectorLogger extends BasicLogger {
 
     @Message(id = 130, value = "Report directory %s does not exist")
     OperationFailedException reportDirectoryDoesNotExist(String reportDirectory);
+
+    @Message(id = 131, value = "Legacy security attribute %s is no longer supported. Please use Elytron configuration instead")
+    OperationFailedException legacySecurityAttributeNotSupported(String attribute);
+
+    @Message(id = 132, value = "Legacy security is no longer supported. Please use Elytron configuration instead")
+    OperationFailedException legacySecurityNotSupported();
```

Comment:
If you change this to OperationFailedRuntimeException it could be thrown from both an OperationStepHandler and from other contexts (like an MSC Service start method) that can properly handle a RuntimeException.

Reply:
I decided to return a String and create exceptions, which match other exceptions thrown, in the code of those classes